### ISSUE TITLE
Unexpected error #9 [BETA]

### ISSUE THREAD

**Issue by EDM115** [EDM115/unzip-bot#11]
```
2022-04-24T22:01:34.921707+00:00 app[worker.1]: Traceback (most recent call last):
2022-04-24T22:01:34.921798+00:00 app[worker.1]:   File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
2022-04-24T22:01:34.921998+00:00 app[worker.1]:     return _run_code(code, main_globals, None,
2022-04-24T22:01:34.922028+00:00 app[worker.1]:   File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
2022-04-24T22:01:34.922136+00:00 app[worker.1]:     exec(code, run_globals)
2022-04-24T22:01:34.922152+00:00 app[worker.1]:   File "/app/unzipper/__main__.py", line 13, in <module>
2022-04-24T22:01:34.922248+00:00 app[worker.1]:     unzipperbot.start()
2022-04-24T22:01:34.922300+00:00 app[worker.1]:   File "/usr/lib/python3.10/site-packages/pyrogram/sync.py", line 66, in async_to_sync_wrap
2022-04-24T22:01:34.922419+00:00 app[worker.1]:     return loop.run_until_complete(coroutine)
2022-04-24T22:01:34.922435+00:00 app[worker.1]:   File "/usr/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
2022-04-24T22:01:34.922649+00:00 app[worker.1]:     return future.result()
2022-04-24T22:01:34.922667+00:00 app[worker.1]:   File "/usr/lib/python3.10/site-packages/pyrogram/methods/utilities/start.py", line 73, in start
2022-04-24T22:01:34.922776+00:00 app[worker.1]:     await self.initialize()
2022-04-24T22:01:34.922791+00:00 app[worker.1]:   File "/usr/lib/python3.10/site-packages/pyrogram/methods/auth/initialize.py", line 46, in initialize
2022-04-24T22:01:34.922904+00:00 app[worker.1]:     self.load_plugins()
2022-04-24T22:01:34.922919+00:00 app[worker.1]:   File "/usr/lib/python3.10/site-packages/pyrogram/client.py", line 617, in load_plugins
2022-04-24T22:01:34.923135+00:00 app[worker.1]:     if plugins[option]:
2022-04-24T22:01:34.923156+00:00 app[worker.1]: KeyError: 'include'
2022-04-24T22:01:35.279338+00:00 heroku[worker.1]: Process exited with status 1
2022-04-24T22:01:35.375084+00:00 heroku[worker.1]: State changed from up to crashed
```

**Comment by EDM115** [comment]
Seems to be a problem with python, idk

**Comment by EDM115** [comment]
That was due to [`PyroGram 2.0`](https://docs.pyrogram.org/releases/v2.0)  
Fixed also qith python 3.9 just in case
